steering_committee.md

```diff
@@ -5,123 +5,57 @@ The FOCUS SC will be responsible for all oversight of the open source projects.
-
-## Steering Committee (SC) meetings
-
-The FOCUS SC will meet at a time and cadence to be determined by the SC. The goal is to meet regularly to address any agenda items quickly and openly.
-
-To attend a FOCUS SC meeting, you must open a PR to add your name to the list of attendees in the meeting agenda. If you have not signed the Membership Agreement you will be prompted to do so. You cannot attend until you have completed this document.
-
-### Scope
-
-The SC provides oversight for all FOCUS efforts within the scope of the project. This includes:
-
-* Approving formal releases of the FinOps Open Cost and Usage Specification (FOCUS).
-* Approving new projects and working groups.
-* Creating, transferring, organizing, and renaming GitHub repos under the [FOCUS GitHub org](https://github.com/FinOps-Open-Cost-and-Usage-Spec).
-* Resolving technical or community issues that span multiple projects, or are deadlocked within a project.
-* Delegating decisions on publishing releases.
-
-The scope of the SC should not include decisions that can be made within one of the projects, unless there is a specific issue that cannot be resolved and requires mediation. In general, the SC prefers that decisions be made at the lowest possible level.
-
-#### Decision-making authority
-
-The following table is a rough outline of who is responsible for decisions. In keeping with the principle that decisions should be made at the lowest possible level and as efficiently as possible, unless there is a specific escalation to the SC.
-
-| Decision | Who decides | Can it be delegated? |
-|---|---|---|
-| Create a new project or working group | SC | No |
-| Approve a Specification release | SC | No |
-| Release a reference implementation | SC or maintainers | Yes |
-| Adding a new repo in the FOCUS organization | SC | No |
-| Changes to this document | SC | No |
-
-
-## Who can participate on the SC
-
-SC meeting are open to anybody who wishes to participate. Aside from items which require a formal vote (such as approving specification verions or elections), anyone who has signed a CLA Membership Agreement may participate in discussions. As defined in the charter, the SC will use the consensus decision-making process as often as possible.
-
-The SC will also include a number of individuals who have been elected as members. These individuals are expected to be active participants on the SC, and to be engaged and responsive on matters that require a formal vote.
-
-You must add yourself to the meeting agenda via PR in order to be able to join a FOCUS meeting.
+The FOCUS SC is governed by the [Charter](FOCUS_-_Membership_Agreement_Package_for_use.pdf), which establishes the Committee and its basic principles and procedures. The charter is designed to provide the SC the freedom to govern itself in an efficient manner. This document establishes SC policies and procedures.
```

Comment:
nit: Should we spell out "SC" in the first reference on this page?

Reply:
Updated to remove the over use of the SC abbreviation